packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -56,14 +57,27 @@ class BuildIOSCommand extends _BuildIOSSubCommand {
   bool get shouldCodesign => boolArg('codesign');
 }
 
-/// Builds an .xcarchive for an iOS app to be generated for App Store submission.
+/// Builds an .xcarchive and optionally .ipa for an iOS app to be generated for
```

Comment:
sounds like it's the other way around now?

Reply:
No, it always builds an .xcarchive, and only builds an IPA if you pass in the plist flag.